filters.go

```diff
@@ -109,6 +109,46 @@ func (p *Pipe) First(lines int) *Pipe {
 	return Echo(output.String())
 }
 
+// Last reads from the pipe, and returns a new pipe containing only the last N
```

Comment:
Note that the methods in this file are in alphabetical order, per the [contributor's guide](https://github.com/bitfield/script/blob/master/CONTRIBUTING.md#style-and-formatting). It just makes it easier to find stuff.